doc/README.hi.md

```diff
@@ -0,0 +1,143 @@
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_03.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_03.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_04.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_04.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_05.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_05.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_06.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_06.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_07.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_07.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_08.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_08.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_09.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_09.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_10.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_10.png)
+[<img src="../fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_11.png" width=405>](../fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_11.png)
+[<img src="../fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_12.png" width=405>](../fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_12.png)
+
+## विवरण
+
+NewPipe ना ही किसी Google फ्रेमवर्क लाइब्रेरी, और ना ही YouTube के API का इस्तेमाल करता है। इस ऐप को चलाने के लिए सिर्फ वेबसाइट्स से जानकारी प्राप्त करने की ज़रूरत है, तो इस ऐप का इस्तेमाल उन डिवाइसों पर भी किया जा सकता है जिनपर Google की सेवाएँ स्थापित नहीं हैं। और NewPipe जैसे कॉपीलेफ्ट किए गए मुक्त सॉफ़्टवेयर का इस्तेमाल करने के लिए आपको YouTube खाते की ज़रूरत नहीं।
+
+### सुविधाएँ
+
+* वीडियो खोजें
+* खाते की ज़रूरत नहीं
+* वीडियो के बारे में साधारण जानकारी पाएँ
+* YouTube के वीडियो देखें
+* YouTube के वीडियो सुनें
+* वीडियो देखते हुए दूसरे ऐप्स का इस्तेमाल करें (फ़्लोटिंग प्लेयर)
+* वीडियो देखने के लिए स्ट्रीम करने का प्लेयर चुनें
+* वीडियो डाउनलोड करें
+* सिर्फ ऑडियो डाउनलोड करें
+* Kodi में वीडियो को खोलें
+* अगले/संबंधित वीडियो देखें
+* YouTube को किसी विशिष्ट भाषा में खोजें
+* उम्र से जुड़ी पाबंदी वाले वीडियी देखें/छिपाएँ
+* चैनलों के बारे में साधारण जानकारी पाएँ
+* चैनल खोजें
+* किसी विशिष्ट चैनल से वीडियो देखें
+* Orbot/Tor के साथ चलाएँ (मुश्किलें आ सकती हैं)
+* 1080p/2K/4K में देखें
+* ऐप आपके देखे गए वीडियो याद रखेगी
+* चैनलों पर सदस्यता लें
+* देखे गए वीडियो खोजें
+* प्लेलिस्ट्स खोजें/देखें
+* प्लेलिस्ट्स को सूची में जोड़कर देखें
+* वीडियों को सूची में डालें
+* खुदकी प्लेलिस्ट्स बनाएँ
+* उपशीर्षक
+* लाइवस्ट्रीम करें/ देखें

```

Comment:
Use only one of the words. Seen together, it sounds like you can not just watch, but also create your own livestreams, which is not the case.